app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java

```diff
@@ -1,13 +1,24 @@
 package com.teamagam.gimelgimel.app.utils;
 
 public class Constants {
-
-    // Just examples. Replace however you need!
-    public static final int HTTP_CONNECT_TIMEOUT =  6000; // milliseconds
-    public static final int HTTP_READ_TIMEOUT = 10000; // milliseconds
+    public static final String MESSAGING_SERVER_URL = "http://ggmessaging-dev1.herokuapp.com";
 
     public static final String build = "debug";
```

Comment:
you have deleted this constant in the release file. choose one - either delete both or keep both. !resolve